packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,276 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { Clipboard, StyleSheet, View } from 'react-native'
+import css from 'src/dev/ValidatorsList.scss'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles } from 'src/styles'
+
+interface ValidatorsListProps {
+  data: any
+  isLoading: boolean
+}
+
+export interface State {
+  expanded?: number
+}
+
+class ValidatorsListApp extends React.PureComponent<ValidatorsListProps & I18nProps, State> {
+  state = {
+    expanded: undefined,
+  }
+
+  cutAddress(address: string) {
+    return address.toUpperCase().replace(/^0x([a-f0-9]{4}).+([a-f0-9]{4})$/i, '0x$1...$2')
+  }
+
+  formatNumber(n: number, decimals: number = Infinity) {
+    return isNaN(+n) ? 'n/a' : (+n).toFixed(decimals).replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  }
+
+  copyAddress(address: string) {
+    Clipboard.setString(address)
+  }
+
+  expand(expanded: number) {
+    if (this.state.expanded === expanded) {
+      this.setState({ expanded: undefined })
+    } else {
+      this.setState({ expanded })
+    }
+  }
+
+  cleanData({ celoValidatorGroups, latestBlock }: any) {
+    const totalVotes: BigNumber = celoValidatorGroups
+      .map(({ votes }) => new BigNumber(votes))
+      .reduce((acc: BigNumber, _) => acc.plus(_), new BigNumber(0))
+
+    return celoValidatorGroups
+      .map(({ account, affiliates, votes }) => {
+        const group = account
+        const gName = group.name
+        return {
+          name: (gName || '').length <= 20 ? gName : `${gName.substr(0, 17)}...`,
```

Comment:
I believe you should be able to do this in CSS https://www.w3schools.com/cssref/css3_pr_text-overflow.asp

Reply:
Only the components that are block-like can apply `text-overflow` property.
There are some ways to fix that:
- Adding to the table the property `table-layout: fixed`, that used the same width for each column. It does the columns that have short content wide.
- Use `table-layout: fixed` and configure the width of some of the params.
- Adding a `div` inside the `td` with a `max-width`, probably better.
I prefer the last one for this case, but anyway I'll see after applying the rest of the changes.